### ISSUE TITLE
Typescript for Typing for GlobalCutModeToggledEventHandler Appears to be wrong?

### ISSUE THREAD

**Issue by cookiekitty** [geoman-io/leaflet-geoman#952]
```
export type GlobalCutModeToggledEventHandler = (e: {
      layer: L.Layer;
      originalLayer: L.Layer;
      shape: PM.SUPPORTED_SHAPES;
   }) => void;
```
Should be
```
export type GlobalCutModeToggledEventHandler = (e: {
      enabled: boolean;
      map: L.Map;
    }) => void;
```

**Comment by Falke-Design** [comment]
Yes you are right, will be fixed 👍🏼

**Comment by Falke-Design** [comment]
Fixed with Release 2.11.3
